packages/vm/src/evm/interpreter.ts

```diff
@@ -261,27 +259,27 @@ export default class Interpreter {
   }
 
   // Returns all valid jump and jumpsub destinations.
-  _getValidJumpDests(code: Buffer): JumpDests {
-    const jumps = []
-    const jumpSubs = []
+  _getValidJumpDests(code: Buffer) {
+    const jumps = new Uint8Array(code.length).fill(0)
 
     for (let i = 0; i < code.length; i++) {
       const curOpCode = this.lookupOpInfo(code[i]).name
 
-      // no destinations into the middle of PUSH
+      // skip over PUSH0-32 since no jump destinations in the middle of a push block
       if (curOpCode === 'PUSH') {
```

Comment:
This still does string comparison which per #1575 should be removed. 
```
    for (let i = 0; i < code.length; i++) {
      const curOpCode = code[i]

      // no destinations into the middle of PUSH
      if (curOpCode >= 0x60 && curOpCode <= 0x7f){
        i += curOpCode - 0x5f
      }
```

Reply:
Good call, fixed.